File: DotNet/CesiumLanguageWriter/Advanced/ICesiumFontValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a CSS font.
    /// </summary>
    public interface ICesiumFontValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value expressed as a CSS font.
        /// </summary>
        /// <param name="font">The font.</param>
        void WriteFont(string font);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumFontValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a CSS font.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumFontValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value expressed as a CSS font.
    * @param font The font.
    */
    void writeFont(String font);
}